iz_importer.py

```diff
@@ -24,7 +24,7 @@ def __init__(self):
 
     def __init__(self, full_import):
         logging.getLogger('PIL').setLevel(logging.ERROR)
-        self.iz_importer_config = read_json_config(collection="IZ")
+        self.iz_importer_config = initialize_collection_config(collection="IZ")
```

Comment:
This change will change all these from "initialize config" (which is a littel weird) to simply "get_config(ID)" which is more typical